os/include/tinyara/pm/pm.h

```diff
@@ -117,8 +101,12 @@ enum pm_domain_e {
  */
 
 #ifndef CONFIG_PM_NDOMAINS
-#define CONFIG_PM_NDOMAINS 1
+#define CONFIG_PM_NDOMAINS 31
 #endif
+/* Registeer Special Domains, which are specific to Kernel*/
+#define PM_IDLE_DOMAIN pm_domain_register("PM_IDLE_DOMAIN")
+#define PM_LCD_DOMAIN  pm_domain_register("PM_LCD_DOMAIN")
+#define PM_BLE_DOMAIN  pm_domain_register("PM_BLE_DOMAIN")
```

Comment:
declaring this definition will allow us not to hardcode PM_IDLE_DOMAIN to 0 and PM_LCD_DOMAIN to 1.

Reply:
The PM should always know the LCD and IDLE domains id(index) for check suspend when state change.

we can use this definition, but we do not need to use it in the PM internal logic.

We can manage it as 0 or 1. We can manage this by registering first like stdin stdout of fd. (This is to avoid having table management logic.)

Reply:
Done